doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -371,9 +371,40 @@ otherwise if `false`:
 * `<targetId>` the internal Neo4j id of target node
 * `<targetLabels>` a list of labels for target node
 
+==== Filter
+
+You can use Spark to filter properties of the relationship, the source node, or the target node. Just use the correct prefix:
+
+If `relationship.node.map` is set to **false**
+
+* ``\`source.[property]` `` for the source node properties
+* ``\`rel.[property]` `` for the relation property
+* ``\`target.[property]` `` for the target node property
+
+If `relationship.node.map` is set to **true**
+
+* ``\`<source>`.\`[property]` `` for the source node map properties
+* ``\`<rel>`.\`[property]` `` for the relation map property
+* ``\`<target>`.\`[property]` `` for the target node map property
+
+in this case, all the map values will be strings, so the filter value must be a string too.
+
+**Note**: when using `relationship.node.map = true` the PushDownFilters support is not active, thus the filters will be applied by Spark and not by Neo4j.
```

Comment:
anche quando si fa una `query` custom, magari spieghiamolo in una `[NOTE]` in appendice quando è supportato e quando no, io tra l'altro aggiungerei una `pushdown.filters.enabled` (default `true`) per eventualmente disabilitarlo e lasciare tutto in carico a Spark

Reply:
usa proprio `[NOTE]`

Reply:
tra l'altro non vedo il riferimento alla `query` :D

Reply:
per favore aggiungi alla documentazione delle option `pushdown.filters.enabled`